grpc.go

```diff
@@ -95,6 +96,25 @@ func onlineResources(resource onlineResource, nbResources int32) (uint32, error)
 			continue
 		}
 
+		if resource.movable {
```

Comment:
It would be great if we had some unit tests for this. I don't think that should block this PR landing as we don't have any unit tests for `onlineResources()` at the moment. I've raised https://github.com/kata-containers/agent/issues/387 so we don't forget about this.